### ISSUE TITLE
Training doesn't progress ?

### ISSUE THREAD

**Issue by hsoreinos** [mikekestemont/DeepScript#3]
Hi,

Thanks for your work. I am trying to reproduce your results with the CLAMM dataset, but after some minor corrections to work with last versions of keras and TF, I encounter several issues. The first one is related to the training step.
I only have one epoch and the accuracy / loss don't vary. Would you have an idea of how I could solve that ?

```-> building dev inputs once:
augmenting train!
   > 0
train.py:124: UserWarning: The `nb_epoch` argument in `fit` has been renamed `epochs`.
  shuffle=True)
Epoch 1/1
360/360 [==============================] - 11s 31ms/step - loss: 1.1237 - accuracy: 0.5167
  curr val acc: 0.5714285714285714
    -> saving model
augmenting train!
   > 0
Epoch 1/1
360/360 [==============================] - 11s 31ms/step - loss: 1.0820 - accuracy: 0.5111
  curr val acc: 0.35714285714285715
augmenting train!
   > 0
Epoch 1/1
360/360 [==============================] - 11s 31ms/step - loss: 1.0603 - accuracy: 0.5389
  curr val acc: 0.35714285714285715
augmenting train!
   > 0
Epoch 1/1
360/360 [==============================] - 11s 31ms/step - loss: 1.0549 - accuracy: 0.5556
  curr val acc: 0.35714285714285715
augmenting train!
   > 0
Epoch 1/1
360/360 [==============================] - 11s 31ms/step - loss: 1.0464 - accuracy: 0.5417
  curr val acc: 0.35714285714285715
augmenting train!
   > 0
Epoch 1/1
360/360 [==============================] - 11s 31ms/step - loss: 1.0522 - accuracy: 0.5417
  curr val acc: 0.35714285714285715
augmenting train!
   > 0
Epoch 1/1
```

Thanks for your help
